src/components/annotations/attributes.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
         valType: 'boolean',
         role: 'info',
         dflt: true,
-        editType: 'calcIfAutorange',
+        editType: 'calcIfAutorange+arraydraw',
```

Comment:
Can you clarify why we now need `arraydraw` here and in `shapes/attributes.js`?

Reply:
`relayout` uses [`containerArrayMatch`](https://github.com/plotly/plotly.js/blob/master/src/plot_api/container_array_match.js) to figure out what edits need to be handled with the special add/remove/edit syntax, and `arraydraw` was just a placeholder for "relayout will handle this, don't worry about it".

Here we're letting the user do their own editing, so there's much less of that complexity. So now I use `editType: 'arraydraw'` as the signal to try and draw components using their `draw` or `drawOne` component methods, hence it has to actually be there!

Reply:
Very clear. Thanks :books: